examples/windows_nodepool_with_private_cluster/gke/backend.tf

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
nit: backends are not needed for examples.

Reply:
Done removed backend.tf file